lib/mongodb-backend.js

```diff
@@ -125,7 +125,7 @@ MongoDBBackend.prototype = {
         values.forEach(function(value){doc[value]=true;});
 
         // update document
-        collection.update(updateParams,{$set:doc},{safe:true,upsert:true},function(err){
+        collection.updateOne(updateParams, {$set:doc}, {w:1, upsert:true}, function(err){
```

Comment:
Are you sure this `update` call is intended for only `one document` and not for `many documents`?

Reply:
why not just use update instead of updateOne?

Reply:
update() is deprecated. I strictly followed the advice in https://mongoosejs.com/docs/deprecations.html to resolve the deprecation warnings and to not change the intended functionality. I used updateOne() because, in this case, update() is only updating the first document found.

Reply:
Seems like a legit change. 
But, don't we need to update `mongodb` version driver in order to use these new functions?